src/Faker/Provider/HtmlLorem.php

```diff
@@ -234,19 +234,19 @@ private function addLoginForm(\DOMElement $element)
 
         $textLabel = $element->ownerDocument->createElement(static::LABEL_TAG);
         $textLabel->setAttribute('for', 'username');
-        $textLabel->textContent = $this->generator->word;
+        $textLabel->textContent = $this->generator->word();
 
         $passwordInput = $element->ownerDocument->createElement(static::INPUT_TAG);
         $passwordInput->setAttribute('type', 'password');
         $passwordInput->setAttribute('id', 'password');
 
         $passwordLabel = $element->ownerDocument->createElement(static::LABEL_TAG);
         $passwordLabel->setAttribute('for', 'password');
-        $passwordLabel->textContent = $this->generator->word;
+        $passwordLabel->textContent = $this->generator->word();
 
         $submit = $element->ownerDocument->createElement(static::INPUT_TAG);
         $submit->setAttribute('type', 'submit');
-        $submit->setAttribute('value', $this->generator->word);
+        $submit->setAttribute('value', $this->generator->word());
 
         $submit = $element->ownerDocument->createElement(static::FORM_TAG);
         $submit->setAttribute('action', $this->generator->safeEmailDomain);
```

Comment:
Add method call here too, on `safeEmailDomain`?

Reply:
Yeah will review the core methods quickly to see what else we hit